nutella/src/container/SignupLoading/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigate } from "react-router-dom";
+import React, { FC, useEffect, useState } from "react";
+import { getOauthSignup } from "../../utils/api/Signup";
+
+const SignupLoadingContainer: FC = () => {
+  const navigate = useNavigate();
+  const code = new URL(window.location.href).searchParams.get("code");
+
+  const onRequest = async () => {
+    try {
+      const signType = (await getOauthSignup(code))?.data.type;
+      console.log(signType);
+      if (signType === undefined) {
+        alert("학교 계정으로 눌러 주세요.");
```

Comment:
이런 alert 전부
```suggestion
     const { showMessage } = useMessageContext();
     showMessage({
        type: "Denial",
        title: "유저 프로필을 가져오는데 실패하였습니다.",
        content: `잠시 후 다시 시도해보거나, 경로를 확인해보세요.`,
      });
```
이런식으로 변경해주세요